### PR TITLE
feat: rating added

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
image

### CODE REVIEW

**Review by SinchanaMS — APPROVED** [review]
No changes to suggest, looks good!

File: component/avatar/avatar.html [hunk 1]
```diff
@@ -72,6 +72,12 @@ <h3 class="list-title">Components</h3>
                 <a href="../modal/modal.html">
                     <li class="list-items">Modal</li>
                 </a>
+                <a href="../rating/rating.html">
+                    <li class="list-items">Rating</li>
+                </a>
+                <a href="../rating/rating.html">
+                    <li class="list-items">Rating</li>
+                </a>
```
Comment (manojsarna) [inline component/avatar/avatar.html:80]: this is repeating , do check , there are multiple links

File: component/rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+            <div class="rating-container">
+                <i class="fas fa-star rating"></i>
+                <i class="fas fa-star rating"></i>
+                <i class="fas fa-star rating"></i>
+                <i class="far fa-star rating"></i>
+                <i class="far fa-star rating"></i>
+            </div>
+
+            <iframe
```
Comment (manojsarna) [inline component/rating/rating.html:133]: its look good , but it would be better if you have added functionality for rating component

**Review by manojsarna — APPROVED** [review]
Do Check the comments

File: component/card/card.html [hunk 3]
```diff
@@ -71,6 +71,9 @@ <h3 class="list-title">Components</h3>
                 <a href="../modal/modal.html">
                     <li class="list-items">Modal</li>
                 </a>
+                <a href="../rating/rating.html">
```
Comment (tarunsankhla) [inline component/card/card.html:74]: Try using direct path instead of ..//../

File: component/rating/rating.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.rating{
+    color: var(--white);
+}
+
+.rating-container{
+    padding: 40px;
```
Comment (tarunsankhla) [inline component/rating/rating.css:6]: Use em instead of px

**Review by tarunsankhla — APPROVED** [review]
Over all the code is great

**Pull request merged**